lib/perl5/site_perl/Utility.pm

```diff
@@ -392,9 +392,10 @@ sub run_whole_pipeline {
 	$dataupload_object->start($args_dict);
 
 	if (!$args_dict->{"no_trim"})  {
-	    print "trim: ", $args_dict->{"no_trim"}," | ", $args_dict->{"only_trim"},"\n";
-	    my $trimming_object = HadoopTrimming->new();
-	    $trimming_object->start($args_dict);
+		print "WARNING: trimming is now done on upload to the HDFS.";
```

Comment:
WARNING sounds very dramatic, but this will just be normal operation in many cases?

Reply:
So what's your suggestion? I use to have `WARNINGS` for possible unexpected behavior in tools.

Anyway, this is happening by default because the `--no-trim` is a switch; Maybe I just change to `Trimming is done on upload to the HDFS` instead, without any warning...

Reply:
Done.